src/tokens/hooks.test.tsx

```diff
@@ -223,8 +223,8 @@ describe('useTokensForSend', () => {
     )
 
     expect(getByTestId('tokenIDs').props.children).toEqual([
-      mockPoofTokenId,
       mockCeloTokenId,
+      mockPoofTokenId,
```

Comment:
just checking that we want to have a different sort order for the send tokens now?

Reply:
This is using the latest order logic with decided. The previous version didn't have any kind of sorting.